src/Carbon/Traits/Date.php

```diff
@@ -1155,6 +1161,11 @@ public function set($name, $value = null)
 
                 break;
 
+            case 'date':
```

Comment:
For safety reasons, I think this case should be postponed until #1992 is fixed, otherwise there will be the same bug for the `date` property.

Reply:
Yep, indeed it's a first try that shouldn't be there.